components/profile/ProfileEditor.js

```diff
@@ -197,6 +197,13 @@ export default function ProfileEditor({
         'You must enter position, institution, and domain information for each entry in your education and career history'
       )
     }
+    if (!profileContent.history.some((p) => !p.end || p.end >= new Date().getFullYear())) {
+      return promptInvalidValue(
+        'history',
+        profile.history?.[0]?.key,
+        'You must enter current education and career info'
```

Comment:
I think we should have a message less confusing because some users might not be studying. 

What about? "You must enter at least one current history record that could be education or career info" @celestemartinez please help us with the wording :)